src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -42,8 +42,17 @@
     <Compile Include="$(CommonPath)\System\Net\HttpValidationHelpers.cs">
       <Link>Common\System\Net\HttpValidationHelpers.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\System\Net\Logging.cs">
-      <Link>Common\System\Net\Logging.cs</Link>
+    <Compile Include="$(CommonPath)\System\Net\Shims\TraceSource.cs">
+      <Link>Common\System\Net\Shims\TraceSource.cs</Link>
```

Comment:
I presume this is needed to support the old Logging.cs dependencies and it will be deleted in the final version?